src/components/AmendmentForm/AmendmentForm.js

```diff
@@ -112,6 +113,54 @@ class AmendmentForm extends React.Component {
     );
   }
 
+  renderPaneFooter() {
+    const {
+      handleSubmit,
+      initialValues,
+      pristine,
+      submitting,
+      invalid
+    } = this.props;
+
+    let id;
+    if (initialValues && initialValues.id) {
+      id = 'clickable-update-amendment';
+    } else {
+      id = 'clickable-create-amendment';
+    }
+
+    const startButton = (
+      <Button
+        buttonStyle="default mega"
+        id="clickable-cancel"
+        marginBottom0
+        onClick={this.props.handlers.onClose}
+      >
+        <FormattedMessage id="stripes-components.cancel" />
+      </Button>
+    );
+
+    const endButton = (
+      <Button
+        buttonStyle="primary paneHeaderNewButton"
```

Comment:
```suggestion
        buttonStyle="primary mega"
```

Should this be `primary mega`? [Asking based on the ui-users example](https://github.com/folio-org/ui-users/blob/master/src/UserForm.js#L274).